target/scripts/helpers/exit.sh

```diff
@@ -0,0 +1,74 @@
+#! /bin/bash
```

Comment:
Does `failure.sh` / `fail.sh` or `error.sh` seem more appropriate than `exit.sh` context wise? I guess it does result in exiting either way though so probably nit-picking :sweat_smile:

Reply:
I will go ahead and renamed it to one of the proposed names :) These names are better suited.